### PR TITLE
iss-hipm-908 Update layout of the download page of routes/exhibits to fit the new design

### PR DESCRIPTION
**Pull request by Senn3**
I have updated the portrait and landscape layout of the ExhibitRouteDownloadPage.

### CODE REVIEW

**Comment by sayantik** [comment]
If you could please resolve the conflicts, then I'll take a look at it

File: Sources/HipMobileUI/ViewModels/Pages/ExhibitRouteDownloadPageViewModel.cs [hunk 1]
```diff
@@ -42,20 +43,19 @@ public ExhibitRouteDownloadPageViewModel(IDownloadable downloadable, IDownloadab
             DownloadableDescription = downloadable.Description;
 
             Message = Strings.DownloadDetails_Text_Part1 + DownloadableName + Strings.DownloadDetails_Text_Part2;
```
Comment (Resudo) [inline Sources/HipMobileUI/ViewModels/Pages/ExhibitRouteDownloadPageViewModel.cs]: Remark from the PO: you should also change the text of message as stated in the description of your task.

**Comment by sayantik** [comment]
newlandscape
newpotrait
oldlandscape
oldpotrait
In the new landscape and the portrait mode the text "Ausschnitt aus dem.. " is not visible completely. You can see the screenshots attached comparing the layouts.

**Comment by Senn3** [comment]
aspectfill landscape
aspectfill
aspectfit landscape
aspectfit
I think we got a problem with the image aspect ratio. One solution is to set the image type to "AspectFit" (see the images above), but this will add some white space at the top and bottom of the image. Maybe we should discuss this topic next monday with the other members.

**Comment by sayantik** [comment]
Sure, we can do that.

**Comment by Senn3** [comment]
I think we should open a new Jira bug, because this problem affects the AppetizerPage, UserRatingPage and the ExhibitRouteDownloadPage. If there and no other problems with this PR it can be approved and merged with the master, right?

**Comment by sayantik** [comment]
@Resudo what do you think about the above?

**Comment by Resudo** [comment]
@sayantik @Senn3 I know about this problem since last semester because I had some problems with it as well. We even have a [task in JIRA](https://atlassian-hip.cs.uni-paderborn.de/jira/secure/RapidBoard.jspa?rapidView=8&projectKey=HIPM&view=planning&selectedIssue=HIPM-739) to find standards for image size and resolution. I will block the bug 914 @Senn3 created with this task since it's unwise to modify the already used images before deciding upon general guidelines for them.

**Pull request merged**